### PR TITLE
Fix assuming tests are run from the project root.

### PR DESCRIPTION
**Pull request by deviantintegral**
When running individual tests in PHPStorm, the cwd is set to the path of the file you are testing. This fixes not being able to resolve `test_services.dist.json`.

### CODE REVIEW

File: tests/bootstrap.php [hunk 1]
```diff
@@ -54,7 +54,7 @@
 }
 
 // Instantiate the service builder
-$aws = Aws\Common\Aws::factory(isset($_SERVER['CONFIG']) ? $_SERVER['CONFIG'] : 'test_services.dist.json');
+$aws = Aws\Common\Aws::factory(isset($_SERVER['CONFIG']) ? $_SERVER['CONFIG'] : (__DIR__) . '/../test_services.dist.json');
```
Comment (GrahamCampbell) [inline tests/bootstrap.php]: you don't need brackets around `__DIR__`

**Comment by mtdowling** [comment]
Looks good. As Graham suggested, we don't need to parens. If it's too much of a hassle, then it doesn't really matter.

**Comment by deviantintegral** [comment]
That was easily the most taxing commit of the day.

**Comment by mtdowling** [comment]
Hahaha :) thanks for the PR!

**Pull request merged**